packages/filesystem/src/common/download/file-download-disposition.ts

```diff
@@ -0,0 +1,140 @@
+/********************************************************************************
+ * Copyright (C) 2019 Bitsler and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+export interface ContentDisposition {
+    parameters: ContentDispositionParams;
+    type: string;
+}
+
+export interface ContentDispositionParams {
+    filename: string;
+    'filename*'?: string;
+    'name'?: string;
+    [propName: string]: string | undefined;
+}
+
+export namespace FileDownloadContentDisposition {
```

Comment:
It should be a class, see https://github.com/theia-ide/theia/wiki/Coding-Guidelines#di-function-export